src/System.Security.Principal.Windows/src/System/Security/Principal/Win32.cs

```diff
@@ -132,7 +132,7 @@ internal static byte[] ConvertIntPtrSidToByteArraySid(IntPtr binaryForm)
 
             try
             {
-                if (TRUE != Interop.mincore.ConvertStringSidToSid(stringSid, out ByteArray))
```

Comment:
Why is deleting it a bad idea?  As far as I can tell, it's currently never used, which means it's dead code.  And if we're concerned about comparing against it, because a true result could actually be any non-zero value, then having it leads to the temptation to use it, which could be a bug.